scrapenews/spiders/dailyvoice.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+from .sitemap import SitemapSpider
+from scrapenews.items import ScrapenewsItem
+from datetime import datetime
+import pytz
+
+SAST = pytz.timezone('Africa/Johannesburg')
+
+
+class DailyVoiceSpider(SitemapSpider):
+    name = 'dailyvoice'
+    allowed_domains = ['www.dailyvoice.co.za']
+
+    sitemap_urls = ['https://www.dailyvoice.co.za/robots.txt']
+    sitemap_follow = [
+        'www.dailyvoice.co.za/news/western-cape',
+        'www.dailyvoice.co.za/news/national',
```

Comment:
I think it's not actually requesting any sites because this is not allowing it to fetch https://www.dailyvoice.co.za/sitemap-1.xml or https://www.dailyvoice.co.za/sitemap-0.xml linked from the sitemap index file https://www.dailyvoice.co.za/sitemap.xml.

[sitemap_follow](https://doc.scrapy.org/en/latest/topics/spiders.html#scrapy.spiders.SitemapSpider.sitemap_follow) is regexes for sitemap URLs, not pages

Reply:
Is the solution therefore to make a crawling spider for the Daily Voice, because the ```<loc>```s in sitemap-0 are not sitemaps themselves?
Let me know if I've misunderstood!